client/src/App.js

```diff
@@ -20,32 +18,10 @@ import './styles/styles.css';
  */
  const App = () => {
  return (
-  <div className="app-container">
```

Comment:
We will leave this up to the front end team, but do you want routing separate from App.js?